README.md

```diff
@@ -287,7 +287,7 @@ To run the workflow on another dataset, you need:
 > - Second column: Path to FASTQ file 1 which should be the forward read for this sample
 > - Third column: Path to FASTQ file 2 which should be the reverse read for this sample
 > 
-> The easiest way to get this file is by using the `generate_samplesheet.sh` script. As input, this script takes a path to raw FASTQ files (`dir_path`), and forward (`forward_suffix`) and reverse (`reverse_suffix`) read suffixes, both of which support regex, and an optional output path (`output_path`). Those using data from s3 should make sure to pass the `s3` parameter. As output, the script generates a CSV file named (`samplesheet.csv` by default), which can be used as input for the pipeline.
+> The easiest way to get this file is by using the `generate_samplesheet.sh` script. As input, this script takes a path to raw FASTQ files (`dir_path`), and forward (`forward_suffix`) and reverse (`reverse_suffix`) read suffixes, both of which support regex, and an optional output path (`output_path`). Those using data from s3 should make sure to pass the `s3` parameter. Those who would like to group samples by some metadata should pass a path to a CSV file containing a header column named `sample,group`, where each row gives the sample name and the group to group by (`group_file`). As output, the script generates a CSV file named (`samplesheet.csv` by default), which can be used as input for the pipeline.
```

Comment:
I'd prefer we say something like "can pass a CSV file [...] or edit the samplesheet manually after generation" (since in cases where the groups CSV isn't readily available manual editing will often be easier).

Reply:
Agreed, updated.